Embedded/MaxMix/MaxMix.ino

```diff
@@ -254,13 +257,36 @@ bool ProcessPackage()
   else if(command == MSG_COMMAND_SETTINGS)
   {
     UpdateSettingsCommand(decodeBuffer, &settings);
-    stateScreen = STATE_DISPLAY_AWAKE;
+    stateDisplay = STATE_DISPLAY_AWAKE;
     return true;
   }
 
   return false;
 } 
 
+
+//---------------------------------------------------------
+//---------------------------------------------------------
+int8_t computeAcceleratedVolume(int8_t encoderDelta, uint32_t deltaTime, int8_t volume) {
```

Comment:
Function names should start with upper case.
Can you rename to ComputeAcceleratedVolume?